File: src/main/java/dqualizer/research/dqapi/repositories/dam/domainstory/ActorRepository.java
package dqualizer.research.dqapi.repositories.dam.domainstory;

import io.github.dqualizer.dqlang.types.dam.domainstory.Actor;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ActorRepository extends MongoRepository<Actor, String> {

}

File: src/main/kotlin/io/github/dqualizer/dqapi/repositories/dam/domainstory/ActorRepository.kt
package io.github.dqualizer.dqapi.repositories.dam.domainstory

import io.github.dqualizer.dqlang.types.dam.domainstory.Actor
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface ActorRepository : MongoRepository<Actor, String>
